doc/in-progress/sui-wallet-with-ledger.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Use Sui Wallet with a Ledger wallet
+---
+
+This guide describes how to use Sui Wallet with your Ledger device. When you connect your Ledger device, you can add up to 10 accounts from your device to Sui Wallet. You can then use the account in Sui Wallet just like any other account, but with the added protection of storing your private keys in a cold storage wallet. This gives you complete control of the private keys for your digital assets. To use your device with Sui Wallet, you just need to install the Sui app on the device, add one or more accounts from the device to Sui Wallet, and then authorize the connection to Sui Wallet when you use the account to perform transactions on the Sui network.
+
+The sections in this guide include:
+* [Requirements](#requirements)
+* [Install the Sui app on your Ledger device](#install-the-sui-app-on-your-ledger-device)
+* [Import accounts from your device to Sui Wallet](#import-accounts-from-your-device-to-sui-wallet)
+* [View account balance](#view-your-account-balance-and-assets)
+* [Receive digital assets using Sui Wallet](#receive-digital-assets-using-sui-wallet)
+* [Send digital assets using Sui Wallet](#send-digital-assets-using-sui-wallet)
+* [Support for Sui Wallet](#support-for-sui-wallet)
+
+## Requirements
+
+Before you connect your device to Sui Wallet:
+* [Install the latest version of Sui Wallet](https://docs.sui.io/devnet/explore/wallet-browser#install-the-sui-wallet-browser-extension) from the Chrome Web Store.
+* [Set up and configure](https://support.ledger.com/hc/en-us/sections/4404369606801-Getting-Started?docs=true) your Ledger device, including updating to the latest firmware.
+* Install [Ledger Live](https://www.ledger.com/ledger-live) and confirm that you can connect your device to Ledger Live.
+
+## Install the Sui app on your Ledger device
+
+You can install the Sui app on your Ledger device from the App Catalog in Ledger Live.
+
+1. Unlock your device.
+1. Open Ledger Live and click **My Ledger** in the left panel.
+1. Press both buttons on the device to allow the secure connection with Ledger.
```

Comment:
Arent there multiple devices? I dont think the nano x has these buttons?

Reply:
the device with only 1 button is no longer available for purchase